docs/dapps/sdk/cli.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 70
+title: CLI
+sidebar_label: CLI
+---
+
+The `CLI` is a tool for creating projects, compiling contracts, and deploying contracts.
+
+## Create a Project
+
+You can create a new project using the `init` command, the `-t` parameter is used to specify the project type:
+
+```shell
+npx @alephium/cli init my-dapp -t react
+```
+
+There are three available project types:
+
+* `base`: Create a Node.js project, this is the default type if the `-t` parameter is not specified
+* `react`: Create a React project
+* `nextjs`: Create a Next.js project
+
+## Configuration
+
+In order to use the `CLI`, you need to have a configuration file, and the default config file is `alephium.config.ts/js` located in the project root directory. The following is an example of a configuration file:
+
+<details>
+<summary>alephium.config.ts</summary>
+<p>
+
+```typescript
+import { Configuration } from '@alephium/cli'
+
+const configuration: Configuration = {
+  // The `networks` field specifies configurations for different networks. It supports three types of networks: devnet, testnet, and mainnet
+  networks: {
+    devnet: {
+      // The `nodeUrl` is the url of the full node
+      nodeUrl: 'http://localhost:22973',
+      // The purpose of private key is for deploying contracts. Since Alephium currently has 4 groups,
+      // the maximum length of `privateKeys` is 4, and each group can have at most one private key.
+      // If you only need to deploy contracts to one group, you only need to specify one private key.
+      privateKeys: ['a642942e67258589cd2b1822c631506632db5a12aabcf413604e785300d762a5'],
+      // The `deploymentFile` field is an optional config. If it is not specified, the default path
+      // is `<project_root>/artifacts/.deployment.<network_id>.json`.
+      deploymentFile: './deployments/.deployment.devnet.json',
+      // The `confirmations` field is used to specify the number of block confirmations to wait for
+      // after contract deployment. This is an optional config. If it is not specified, it defaults
+      // to 1 for devnet and 2 for testnet and mainnet.
+      confirmations: 1
+    }
+  }
+}
+
+// You must export the `configuration` from the config file
+export default configuration
+```
+
+</p></details>
+
+In most cases, you only need to specify the `networks` config, and the other optional configs can work well by default. But you can also configure these optional configs if needed:
```

Comment:
We have a config for the directory of deployment files, right?

Reply:
This config is in the network settings: https://github.com/alephium/docs/pull/374/files#diff-4d2fb94ca21bcfde025521c050d7307d3c79157ecfb41e577cbe0746e181e607R46